### DISCUSSION TITLE
BG Color in cells that span rows not working correctly across pages

### DISCUSSION THREAD

**Discussion by llarsen71** [lvjr/tabularray#441] [Q&A]
I have a page with cells that span rows. The cell that spans rows has a background color and breaks across pages. The cell color goes outside the bounds of the cell on the first page and is not part of the cell on the second. Is there any way to fix this aside from not using cell color?

```latex
\documentclass[letterpaper,10pt]{article}
\usepackage{tabularray}
\usepackage{xcolor}

%===============================================================================
\begin{document}
%===============================================================================
Test page
\vspace{6in}

\begin{longtblr}{hlines, vlines, colspec={ll}}
1  & \SetCell[r=9]{bg=blue7} This is a test of spanning color across pages \\
2  &                             \\
3  &                             \\
4  &                             \\
5  &                             \\
6  &                             \\
7  &                             \\
8  &                             \\
9  &                             \\
\end{longtblr}

\end{document}
```
First page shown below:

![image](https://github.com/lvjr/tabularray/assets/1498201/4a994f34-6dac-4d73-aca4-386b7e1120c9)

Second page:

![image](https://github.com/lvjr/tabularray/assets/1498201/adf3bddc-4d29-4ff4-8a81-3492b8023652)

**Comment by muzimuzhi** [comment]
Hmm allowing page breaks in between "merged cells" is not a good idea. Note the cell content "This is a test of spanning color across pages" is typeset outside of the first partial table too.

**Reply by yatescr** [reply]
How would you disable such page breaks? It seems to me the onus is on the package to prevent such page breaks.
